desc/compute/_geometry.py

```diff
@@ -22,6 +22,34 @@ def _V(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="V",
```

Comment:
Now there are two difference compute functions both with the name `"V"`. I'm confused about when one is called over the other. Does this new one only get called for `FourierRZToroidalSurface`, and the original one gets called for all other classes? But this new one is actually more general, where as the original one will only work for an `Equilibrium`.

Reply:
If the new compute function is valid in all use cases, can we replace the old one instead of having two different functions that compute the same quantity? 

If we need to keep both implementations, we should add a test to check them against each other.

Reply:
The new one is more general and could replace the old one, I'll make that change.

Reply:
actually its a bit tricky. The new compute function is only correct if the grid has nodes along the LCFS, while the original one assumes quadrature nodes in the volume, which genreally are a bit short of the LCFS. in `eq.compute` we use a quadrature grid to compute everything with `dim==0`, but that will give an incorrect result with the new compute method (`Surface.compute` uses a linear grid so its correct using the new method)

Reply:
OK. It still seems messy to have redundant compute methods. Should we add a `parameterization` to the old method to specify it works for `Equilibrium`? We need something to document when to use one over the other.

Reply:
the default parameterization is Equilibrium unless otherwise specified. I don't think its much of an issue since users will never really interact with these functions directly.